### PR TITLE
Adding copyright notice

### PR DESCRIPTION
**Pull request by usbsnowcrash**
This adds the copyright notices and also runs rubocop across to correct whitespace and tabbing type issues.  This one should be merged last after the other copyright notice ones.

### CODE REVIEW

**Comment by helious** [comment]
switching to shorthand over hash rockets lowers this gem's compatibility with older versions of ruby - do we care?

**Comment by jusroberts** [comment]
Haha, technically a major version change!

**Comment by usbsnowcrash** [comment]
@helious do we support pre 1.93?

**Comment by helious** [comment]
not on our apps, right?
it was a point of contention before when this was considered an open-source ruby CB api wrapper. if that's no longer a concern of ours, nbd. just bringing it back up

**Comment by ausmith** [comment]
Put in the README that it's 2.0+?

File: lib/cb/clients/talent_network.rb [hunk 1]
```diff
@@ -13,39 +22,38 @@ def self.join_form_branding(tndid)
...
         my_api.append_api_responses(json_hash, json_hash)
       end
 
-      def self.tn_job_information(job_did, join_form_intercept="true")
+      def self.tn_job_information(job_did, join_form_intercept = 'true')
         my_api = Cb::Utils::Api.instance
-        json_hash = my_api.cb_get("#{Cb.configuration.uri_tn_job_info}/#{job_did}/json", :query=> {
-           :RequestJoinFormIntercept => join_form_intercept})
+        json_hash = my_api.cb_get("#{Cb.configuration.uri_tn_job_info}/#{job_did}/json", query: {
+                                    RequestJoinFormIntercept: join_form_intercept })
```
Comment (ausmith) [inline lib/cb/clients/talent_network.rb]: I think rubocop got the tabbing wrong here.

**Comment by usbsnowcrash** [comment]
Fixing tabbing, adding note about ruby versions

File: lib/cb/models/implementations/category.rb [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
     class Category
       attr_accessor :code, :name, :language
-      def initialize(args={})
-        @code					= args["Code"]              || String.new
-        @name					= args["Name"]["#text"]     || String.new
-        @language     = args["Name"]["@language"] || String.new
+      def initialize(args = {})
+        @code					= args['Code'] || ''
+        @name					= args['Name']['#text'] || ''
+        @language     = args['Name']['@language'] || ''
```
Comment (ausmith) [inline lib/cb/models/implementations/category.rb]: Removes the extra padding in some places and not others, huh.

File: lib/cb/models/implementations/education.rb [hunk 3]
```diff
@@ -9,8 +19,8 @@ def initialize(args = {})
         @code          = args['Code'] || ''
         @language			 = args['Name']['@language'] unless args['Name'].nil?
         @text          = args['Name']['#text'] unless args['Name'].nil?
-        @language      ||= ''
-        @text          ||= ''
+        @language ||= ''
+        @text ||= ''
```
Comment (ausmith) [inline lib/cb/models/implementations/education.rb]: Oh, I guess it does this spacing removal when there's a `||` in front of the equal sign, but not when it's a plain equal. Interesting.

**Comment by ausmith** [comment]
@usbsnowcrash only the lack of a CHANGELOG entry is holding up this final copyright PR.

**Comment by usbsnowcrash** [comment]
Added changelog entry

**Comment by ausmith** [comment]
Yup, was awaiting the build.

**Pull request merged**